oojs.js

```diff
@@ -1,4 +1,27 @@
 // Define a Playlist class. It should have...
+class Playlist {
+  constructor(title){
+    this.title = title;
+    this.songs = [];
+    this.favorites = 0;
+  }
+  addSong(song){
+    this.songs.push(song);
+   }
+ }
+
+ let newPlaylist = new Playlist("Curtis Sweet Jams");
+
+ class Album extends Playlist {
+  constructor(artist,title,songs,favorites) {
+    super(title,songs,favorites);
```

Comment:
This won't give an error, but since your Playlist constructor only defines `title`, the two extra parameters will be thrown away when super is called and not be used to set songs or favorites.